<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>classe</class>
 <widget class="QMainWindow" name="classe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#menuWindow {
background:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

#mainFrame {
border: 3px solid gray;
border-radius: 40px;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}
QLineEdit {
padding: 2px;
border-style: solid;
border:2px solid gray ;

border-radius: 8px;


}

QDateEdit 
{
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}


QPushButton {
color: white;

background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 13px;
padding-left: 5px;
padding-right: 5px;

}


QLabel {
color :  QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
font-weight: bold;
font-size: 15px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="retour">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>528</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#retour {
background-color: transparent;
border-image: url(:/img/img/retour.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ajouter_4">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>448</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_ajouter_4 {
background-color: transparent;
border-image: url(:/img/img/store.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>               Parametres    </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_salle">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>338</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_salle {
background-color: transparent;
border-image: url(:/img/img/classe.jpg);
background: none;
border: none;
background-repeat: none;
color : gray ;

font-size: 20px;
}</string>
    </property>
    <property name="text">
     <string>Salle</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#label {
background-color: transparent;
border-image: url(:/img/img/espace etudiant.jpg);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>871</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>651</width>
        <height>621</height>
       </rect>
      </property>
      <property name="title">
       <string>Classe</string>
      </property>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_Salle">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Numero</string>
       </property>
      </widget>
      <widget class="QTableView" name="tabsalles">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>401</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>130</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Effectif</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>320</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Batiment</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_nom_batiment">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>494</x>
         <y>420</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>320</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Salle</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Batiment</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>171</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Actualiser/Ordonner</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>360</y>
         <width>591</width>
         <height>51</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>- - - - - - - - - -   Eteindre Toutes Les Salles  - - - - - - - - - -</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_historique">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>170</y>
         <width>401</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;   &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;  En Attente d'instructions...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_numero_salle">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser_Batiment">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>310</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_effectif">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>171</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>811</width>
        <height>391</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(175, 203, 255);</string>
        </property>
        <property name="text">
         <string>Allumer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(194, 179, 179);</string>
        </property>
        <property name="text">
         <string>Eteindre</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>421</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>Intensite</string>
        </property>
        <widget class="QSlider" name="horizontalSlider">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>160</width>
           <height>22</height>
          </rect>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);</string>
         </property>
         <property name="singleStep">
          <number>20</number>
         </property>
         <property name="sliderPosition">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>30</y>
          <width>291</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>GroupBox</string>
        </property>
        <widget class="QSlider" name="verticalSlider_2">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>50</y>
           <width>22</width>
           <height>160</height>
          </rect>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="sliderPosition">
          <number>70</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>30</y>
           <width>181</width>
           <height>191</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-image: url(:images/volets.png);
border: none;
}
</string>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </widget>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Manuel</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>150</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Automatique</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Heure_temp">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>200</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Horaire actuelle</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string/>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_id_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>60</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Identifiant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>180</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Pr√©nom</string>
        </property>
       </widget>
       <widget class="QTableView" name="tabetudiant_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>50</y>
          <width>501</width>
          <height>181</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supprimer_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser_historique_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>230</y>
          <width>501</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 0, 0);</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;   &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;  En Attente d'instructions...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter_7">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>280</y>
          <width>121</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Ordre Alphabetique</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>610</x>
          <y>280</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_faire_appel">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Appel</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <zorder>label</zorder>
   <zorder>retour</zorder>
   <zorder>pb_ajouter_4</zorder>
   <zorder>pushButton_salle</zorder>
   <zorder>stackedWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
